webapp/channels/src/components/admin_console/filter/team_filter_dropdown/team_filter_checkbox.tsx

```diff
@@ -8,7 +8,7 @@ type Props = {
     updateOption: (checked: boolean, name: string) => void;
 };
 
-const TeamFilterCheckbox: React.FC<Props> = ({
+const TeamFilterCheckbox: React.FC<Props> = React.memo(({
```

Comment:
nit: we prefer the `React.memo` in the `export default` line to keep component declaration cleaner, that is normally a more complicated part.